### PR TITLE
Update Installation.md

### PR DESCRIPTION
**Pull request by unityjeffrey**

### CODE REVIEW

**Comment by chriselion** [comment]
This is already linked from https://github.com/Unity-Technologies/ml-agents/tree/master/docs#deprecated-docs - do we really need to add another place where we say "we don't really support this but..."?

**Comment by unityjeffrey** [comment]
i believe so.  many of the first time users walk through the installation doc as a first step.  Since its unlikely they will see the depricated docs sections, its important we call it out there as we don't want to an abandonment of the funnel

**Comment by unityjeffrey** [comment]
i made this change as I was walking some folks at Neurips through the workflow and couldn't find it.

File: docs/Installation.md [hunk 1]
```diff
@@ -21,6 +21,9 @@ We now support a single mechanism for installing ML-Agents on Mac/Windows/Linux
 Environments. For more information on Virtual Environments and installation instructions,
 follow this [guide](Using-Virtual-Environment.md).
 
+Although we don't support Anaconda installation path of ML-Agents for Windows, the previous guide
+is still in the docs folder.  Please refer to [Windows Installation (Depricated)](Installation-Windows.md).
```
Comment (chriselion) [inline docs/Installation.md]:
```suggestion
is still in the docs folder.  Please refer to [Windows Installation (Deprecated)](Installation-Windows.md).
```

**Pull request merged**